src/igvfd/tests/test_types_file.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+import webtest
```

Comment:
I don't think you need this, but, if you do, should import in the test's local scope.